scripts_future_API/run_stokes3D.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+module load LUMI/22.08
+module load partition/G
+module load rocm/5.3.3
+
+export MPICH_GPU_SUPPORT_ENABLED=1
+
+export LD_PRELOAD=${CRAY_MPICH_ROOTDIR}/gtl/lib/libmpi_gtl_hsa.so
```

Comment:
This could be skipped if you have following two last lines in global prefs for LocalPrefs:
```
[AMDGPU]
use_artifacts = false

[MPIPreferences]
_format = "1.0"
abi = "MPICH"
binary = "system"
libmpi = "libmpi_cray"
mpiexec = "srun"
preloads = ["libmpi_gtl_hsa.so"]
preloads_env_switch = "MPICH_GPU_SUPPORT_ENABLED"
```

Reply:
Cool, didn't know this, thanks!